package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ruff",
   "displayName": "Ruff",
   "description": "A Visual Studio Code extension with support for the Ruff linter.",
-  "version": "2024.16.0",
+  "version": "2024.17.0",
```

Comment:
This should be bumped to an _odd_ version to ensure that it's published to the pre-release channel. Here's an example of what it should look like: https://github.com/astral-sh/ruff-vscode/pull/388/files.

Reply:
(also read Charlie's comment and got really confused lol)

Reply:
Sorry, yes, I was tired, it needs the `-dev` suffix though.